src/components/ProofOfAddress/Guidance/style.css

```diff
@@ -0,0 +1,57 @@
+@import (less) "../../Theme/constants.css";
```

Comment:
getting closer and closer to getting a name clash to css classes. I have created a PR to mitigate this issue. https://github.com/onfido/onfido-sdk-ui/pull/415/files